src/Layouts/stars/feed/starActivity.tsx

```diff
@@ -15,40 +16,36 @@ const StarActivity = ({ activity }: activityProps) => {
       </div>
     );
   }
+
+  const activityDates = activity
```

Comment:
```
  const activityDates = activity
    .map((log) => log.time.getTime())
    .reduce((alldates, date, i, array) => {
      array.indexOf(date) === i && alldates.push(new Date(date));
      return alldates;
    }, [] as Date[]);
```

Reply:
not tested 100% but need to work(may be some empty array shit but its bs

Reply:
I did it, but why is this better than before?